jolt-core/src/jolt/trace/rv.rs

```diff
@@ -930,14 +937,19 @@ impl JoltProvableTrace for RVTraceRow {
             | RV32IM::BGE
             | RV32IM::BLTU
             | RV32IM::BGEU
-            | RV32IM::JAL
+            | RV32IM::JAL 
             | RV32IM::JALR
             | RV32IM::LUI => false,
             _ => true,
         };
 
         flags[7] = match self.opcode {
-            RV32IM::ADD | RV32IM::ADDI | RV32IM::JAL | RV32IM::JALR | RV32IM::AUIPC => true,
+            RV32IM::ADD 
+            | RV32IM::ADDI 
+            // Arasu: don't count these here. 
```

Comment:
Nit: Make this comment `TODO(github-handle)` that way we can `cmd + F` these later and ensure they get converted to tracking bugs (linear) or removed. We don't want comments directed at an individual at time of publish.